styles/style.css

```diff
@@ -0,0 +1,236 @@
+    text-align: center;
+    margin-top: 30px;
+    margin-bottom: 30px;
+    margin-left: 32px;
+}
+.logo2{
+    display: inline-block;
+    width: 3px;
+    height: 16px;
+    background-color: white;
+    border-radius: 20px;
+    background: linear-gradient(to bottom right,#E65C00,#F9D423);
+    text-align: center;
+    margin-top: 32px;
+    margin-bottom: 32px;
+    margin-left: 5px;
+}
+.logo3{
+    display: inline-block;
+    width: 3px;
+    height: 12px;
+    background-color: white;
+    border-radius: 15px;
+    background: linear-gradient(to bottom right, #00D2FF, #3A7BD5);
+    text-align: center;
+    margin-top: 34px;
+    margin-bottom: 34px;
+    margin-left: 5px;
+}
+.logo4{
+    display: inline-block;
+    text-align: center;
+    position: absolute;
+    left: 71px;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 22px;
+    margin-top: 27px;
+    margin-bottom: 27px;
+    margin-left: 5px;
+}
+.logo5{
+    display: inline-block;
+    text-align: center;
+    position: absolute;
+    left: 114px;
+    font-weight: 700; 
+    font-style: normal;
+    font-size: 22px;
+    margin-top: 27px;
+    margin-bottom: 27px;
+    margin-right: 71px;
+}
+
+
+.header-menu_item{
+    display: inline-block;
+    position: absolute;
+    color: black;
+    font-size: 12px;
+    font-weight: 700px;
+    line-height: 18px;
+    text-transform: uppercase;
+    padding: 30px;
+}
+.about{
+    left: 219px;
+}
+.features{
+    left: 302px;
+}
+.pricing{
+    left: 404px;
+}
+.gallery{
+    left: 494px;
+}
+.team{
+    left: 584px;
+}
+.fa1{
+    right: 146px;
+}
+.fa2{
+    right: 93px;
+}
+.fa3{
+    right: 35px;
+}
+
+/* Title */
+
+.tittle{
+    position: absolute;
+    width: 518px;
+    height: 61px;
+    left: 461px;
+    top: 200px;
+    font-family: Roboto;
+    font-style: normal;
+    font-weight: bold;
+    font-size: 52px;
+    line-height: 61px;
+    text-align: center;
+    color: #FFFFFF;
+}
+.subtittle{
+    position: absolute;
+    width: 536px;
+    height: 21px;
+    left: 452px;
+    top: 283px;
+    font-family: Roboto;
+    font-style: normal;
+    font-weight: normal;
+    font-size: 18px;
+    line-height: 21px;
+    text-align: center;
+    color: #FFFFFF;
+}
+.button{
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    align-items: center;
+    padding: 5px 26px;
+    
+    position: absolute;
+    width: 200px;
+    height: 50px;
+    left: 620px;
+    top: 350px;
+    
+    background: #FFFFFF;
+    border: 1px solid #000000;
+    box-sizing: border-box;
+    border-radius: 5px;
+}
+.button-text{
+    position: static;
+    width: 200px;
+    height: 50px;
+    left: 64.5px;
+    top: 15.5px;
+    font-family: Roboto;
+    font-style: normal;
+    font-weight: normal;
+    font-size: 16px;
+    line-height: 19px;
+    text-align: center;
+    color: #000000;
+    flex: none;
+    order: 0;
+    flex-grow: 0;
+    margin: 0px 10px;
+}
+.elips79{
```

Comment:
remove all this styles, for ellipses, you can do it easier, without these left, top and other properties